WORKSPACE.bazel

```diff
@@ -46,14 +46,14 @@ protobuf_deps()
 # rules_proto for the proto_library rule
 #
 http_archive(
-  name = "rules_proto",
-  sha256 = "6a0b76595cb8bb97cec085c6b45c24337d30615d636f4941225c05f017690b6f",
-  strip_prefix = "rules_proto-25c8af9d56980df7732b2f8639c5d43ef878a6f7",
-  urls = [
-    "https://mirror.bazel.build/github.com/bazelbuild/rules_proto/archive/25c8af9d56980df7732b2f8639c5d43ef878a6f7.tar.gz",
-    "https://github.com/bazelbuild/rules_proto/archive/25c8af9d56980df7732b2f8639c5d43ef878a6f7.tar.gz",
-  ],
+    name = "rules_proto",
+    sha256 = "903af49528dc37ad2adbb744b317da520f133bc1cbbecbdd2a6c546c9ead080b",
+    strip_prefix = "rules_proto-6.0.0-rc0",
```

Comment:
Can we use the latest stable release instead? If not, please could you add a comment? 

[6.0.0-rc2](https://github.com/bazelbuild/rules_proto/releases/tag/6.0.0-rc2) exists. If an rc version is needed, can we use the latest rc instead?

Reply:
The latest stable is much too old. I'll bump to rc2 (needed to add another dep for it to work).

Reply:
Added a comment!